include/sisl/grpc/rpc_client.hpp

```diff
@@ -166,21 +166,20 @@ class ClientRpcDataFuture : public ClientRpcDataInternal< ReqT, RespT > {
 class GenericClientResponse {
 public:
     GenericClientResponse() = default;
-    GenericClientResponse(grpc::ByteBuffer const& buf);
+    GenericClientResponse(grpc::ByteBuffer const& buf) : m_response_buf{buf} {}
```

Comment:
I'm surprised this isn't marked `explicit`; is that intentional?

Reply:
no, just an oversight. We can add the explicit specifier.